src/components/productdisplayitem.main.jsx

```diff
@@ -430,6 +435,9 @@ class ProductDisplayItemMain extends React.Component {
           availabilityString = intl.get('out-of-stock');
         }
       }
+      const productTitle = productData._definition[0]['display-name'];
+      const productDescription = productData._definition[0].details ? (productData._definition[0].details.find(detail => detail['display-name'] === 'Summary')).value : '';
```

Comment:
This doesn't actually fetch the description summary. Take a look at the `renderAttributes()` function in this class for info on grabbing the value.

It's returned as `detail['display-value']` of the `detail` from your find